build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 group   = "org.veupathdb.lib"
-version = "8.0.0"
+version = "8.1.0"
```

Comment:
I would think this would be a breaking change since the return types of the UserProvider methods are changing in a way that will cause compile errors anywhere that we unwrap the option to use in an explicitly `User` typed context.